spec/systems/keywords/list_keywords_spec.rb

```diff
@@ -0,0 +1,54 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+describe 'View keywords list', type: :system do
+  context 'given NO keyword' do
+    it 'displays a no-keyword notice and DO NOT displays the pagination nav' do
+      Keyword.delete_all
```

Comment:
As commented in other PRs, this can be safely removed

Reply:
Fixed in 4c34b3e07c7d5c18ff6255687b4f66ae63c55737